contrib/babelfishpg_tsql/src/fts_parser.y

```diff
@@ -176,13 +201,11 @@ static char *trim(char *s) {
     size_t end;
     size_t newLength;
 
-    if (s == NULL) {
-        return NULL;
-    }
-
     length = strlen(s);
```

Comment:
strlen() will fail on a NULL input - you need to check for NULL before calling this